nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -994,6 +1031,257 @@ After going through this checklist, you're ready to go!
+				addtimer(CALLBACK(src, .proc/eject), 10 SECONDS)
+				ejecting = TRUE
+				return
+			else
+				to_chat(usr, "<span class='notice'>WARNING AUTO-EJECT SEQUENCE CANCELLED.</span>")
+				relay('nsv13/sound/effects/fighters/switch.ogg')
+				ejecting = FALSE
+				return
+
+		if("docking_mode")
+			to_chat(usr, "<span class='notice'>You [docking_mode ? "disengage" : "engage"] [src]'s docking computer.</span>")
+			docking_mode = !docking_mode
+			relay('nsv13/sound/effects/fighters/switch.ogg')
+			return //Dodge the cooldown because these actions should be instant
+		if("brakes")
+			toggle_brakes()
+			relay('nsv13/sound/effects/fighters/switch.ogg')
+			return //Dodge the cooldown because these actions should be instant
+		if("weapon_safety")
+			toggle_safety()
+			relay('nsv13/sound/effects/fighters/switch.ogg')
+			return //Dodge the cooldown because these actions should be instant
+	warmup_cooldown = TRUE
+	addtimer(VARSET_CALLBACK(src, warmup_cooldown, FALSE), 3 SECONDS)
+	relay('nsv13/sound/effects/fighters/switch.ogg')
+
+/obj/structure/overmap/fighter/proc/get_fuel()
+	var/obj/item/twohanded/required/fighter_component/fuel_tank/sft = get_part(/obj/item/twohanded/required/fighter_component/fuel_tank)
+	if(!sft)
+		return 0
+	var/return_amt = 0
+	for(var/datum/reagent/aviation_fuel/F in sft.reagents.reagent_list)
+		if(!istype(F))
+			continue
+		return_amt += F.volume
+	return return_amt
+
+/obj/structure/overmap/fighter/proc/set_master_caution(state)
+	var/master_caution = state
+	if(master_caution)
+		relay('nsv13/sound/effects/fighters/master_caution.ogg', "<span class='warning'>WARNING: Master caution.</span>", loop=FALSE, channel=CHANNEL_BUZZ)
+	else
+		stop_relay(CHANNEL_BUZZ)
+
+/obj/structure/overmap/fighter/proc/use_fuel()
+	if(flight_state < APU_SPUN) //No fuel? don't spam them with master cautions / use any fuel
+		return FALSE
+	var/amount = fuel_consumption
+	var/obj/item/twohanded/required/fighter_component/fuel_tank/sft = get_part(/obj/item/twohanded/required/fighter_component/fuel_tank)
+	if(!sft)
+		flight_state = NO_FUEL
+		set_master_caution(TRUE)
+		return FALSE
+	sft.reagents.remove_reagent(/datum/reagent/aviation_fuel, amount)
+	if(get_fuel() >= amount)
+		set_master_caution(FALSE)
+		return TRUE
+	if(flight_state < NO_FUEL) //Stops people from getting spammed
+		flight_state = NO_FUEL
+		set_master_caution(TRUE)
+	return FALSE
+
+/obj/structure/overmap/fighter/proc/empty_fuel_tank()//Debug purposes, for when you need to drain a fighter's tank entirely.
+	var/obj/item/twohanded/required/fighter_component/fuel_tank/sft = get_part(/obj/item/twohanded/required/fighter_component/fuel_tank)
+	if(!sft)
+		return FALSE
+	sft.reagents.clear_reagents()
+	say("Fuel tank emptied!")
+
+/obj/structure/overmap/fighter/proc/get_max_fuel()
+	var/obj/item/twohanded/required/fighter_component/fuel_tank/sft = get_part(/obj/item/twohanded/required/fighter_component/fuel_tank)
+	if(!sft)
+		return 0

```

Comment:
FALSE

Reply:
It should return a max of 0 rather than FALSE. I guess they evaluate to the same thing but that makes no sense to me.

Reply:
Its easier to understand at a glance.

Reply:
FALSE is not a unit of volume - this doesn't evaluate to false, it evaluates to 0 liters/gallons/units